src/main/webapp/WEB-INF/pages/TrialManager/createTrial.html

```diff
@@ -807,8 +783,12 @@ <h1 class="page-header col-xs-12 edit-trial-page-identifier">
 	<script type="text/javascript" th:src="@{/static/js/trialmanager/studyInstanceService.js}"></script>
 	<script type="text/javascript" th:src="@{/static/js/trialmanager/environments.js}"></script>
 	<script type="text/javascript" th:src="@{/static/js/trialmanager/experimentalDesign.js}"></script>
+	<script type="text/javascript" th:src="@{/static/js/trialmanager/saveList.js}"></script>
 	<script type="text/javascript" th:src="@{/static/js/trialmanager/r-package/r-package.js}"></script>
 	<script type="text/javascript" th:src="@{/static/js/trialmanager/visualization/visualization.js}"></script>
+	<script type="text/javascript" th:src="@{/static/js/trialmanager/germplasm-study-source/GermplasmStudySource.js}"></script>
```

Comment:
Should the file names of these new JS files be camel case (ie. start with lower case)?